src/include/kdbprivate.h

```diff
@@ -539,6 +539,54 @@ int elektraKeyLock (Key * key, enum elektraLockOptions what);
 
 ssize_t ksSearchInternal (const KeySet * ks, const Key * toAppend);
 
+/**
```

Comment:
I am aware of the embedded systems binary size, but I thought even they could also use this code.
I will move it to another header file.